packages/samples/angular/project.json

```diff
@@ -6,7 +6,10 @@
       "dependsOn": ["^build", "clean"],
       "executor": "nx:run-commands",
       "options": {
-        "commands": ["npm run build:assets", "npm run build:angular"],
+        "commands": [
```

Comment:
Since angular/nextjs can't handle url imports I've disabled their build when building the CDN resources.

Reply:
They can't handle url by default: https://nextjs.org/docs/app/api-reference/next-config-js/urlImports

The thing is it does not matter since our samples use the NPM packages, not the CDN.

This issue will only happen when using the package from the CDN. So if a user decides to import the package from the CDN instead of from NPM in a nextjs implementation, they would have to enable urlImports anyway!